### PR TITLE
Using `to_value` instead of `to(...).value`

### PR DESCRIPTION
**Pull request by Yash-10**
I think we would agree upon this, it's easier to write and slightly faster too.

Thanks!

### CODE REVIEW

File: src/poliastro/core/earth_atmosphere/util.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def _get_index(x, x_levels):
     """
     for i, value in enumerate(x_levels):
         if i < len(x_levels) and value < x:
-            pass
+            continue
```
Comment (astrojuanlu) [inline src/poliastro/core/earth_atmosphere/util.py:96]: ✔️

**Review by astrojuanlu — APPROVED** [review]
Thanks a lot! Must have taken some time to go through all these :)

File: src/poliastro/threebody/flybys.py [hunk 2]
```diff
@@ -34,11 +34,11 @@ def compute_flyby(v_spacecraft, v_body, k, r_p, theta=0 * u.deg):
         Turn angle.
 
     """
-    v_spacecraft = v_spacecraft.to(u.km / u.s).value
-    v_body = v_body.to(u.km / u.s).value
-    k = k.to(u.km ** 3 / u.s ** 2).value
-    r_p = r_p.to(u.km).value
-    theta = theta.to(u.rad).value
+    v_spacecraft = v_spacecraft.to_value(u.km / u.s)
+    v_body = v_body.to_value(u.km)
```
Comment (astrojuanlu) [inline src/poliastro/threebody/flybys.py:38]: Woops!

```suggestion
    v_body = v_body.to_value(u.km / u.s)
```

Comment (Yash-10) [reply src/poliastro/threebody/flybys.py]: :sweat_smile:

**Pull request merged**